manifests/init.pp

```diff
@@ -56,32 +56,26 @@
   String  $config_file_owner                     = 'root',
   String  $config_file_group                     = 'root',
   Stdlib::Filemode $config_file_mode             = '0644',
-  Hash    $receivers                             = {
-    'otlp' => {
-      'protocols' => {
-        'http' => {},
-        'grpc' => {},
-      },
-    },
-  },
-  Variant[Hash,String[1]]    $processors                            = {},
-  Variant[Hash,String[1]]    $exporters                             = {},
-  Variant[Hash,String[1]]    $pipelines                             = {},
-  Hash    $extensions                            = {},
+  Hash[String, Hash] $receivers = {},
+  Hash[String, Hash]     $processors                            = {},
+  Hash[String, Hash]     $exporters                             = {},
+  Hash[String, Hash]     $pipelines                             = {},
+  Hash[String, Hash]    $extensions                            = {},
```

Comment:
This is inconsistent. I would either align parameters, or (my personal preference) stop aligning.
```suggestion
  Hash[String, Hash] $processors = {},
  Hash[String, Hash] $exporters = {},
  Hash[String, Hash] $pipelines = {},
  Hash[String, Hash] $extensions = {},
```